reana_job_controller/config.py

```diff
@@ -66,7 +66,7 @@
 
 
 VOMSPROXY_CONTAINER_IMAGE = os.getenv(
-    "VOMSPROXY_CONTAINER_IMAGE", "docker.io/reanahub/reana-auth-vomsproxy:1.2.0"
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏This PR is not really needed, since upgrade to 1.3.0 is already done on the `maint-0.9` branch, and will come to the `master` branch during maint-to-master upstream merge. Let's rather do that, since it'll bring PUNCH backend support etc.